addons/test_website/static/tests/field_html_file_upload.js

```diff
@@ -1,21 +1,25 @@
 /** @odoo-module **/
 
 import { FileSelectorControlPanel } from '@web_editor/components/media_dialog/file_selector';
-import { getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
+import { click, getFixture, makeDeferred, patchWithCleanup } from "@web/../tests/helpers/utils";
```

Comment:
`test_website` purpose is to add python tests which would require a module update / install /uninstall, or that would create views/controllers/models for test purpose only, where we don't want to add fake data in `website` module.

See the manifest
https://github.com/odoo/odoo/blob/09736d81c844b1caa1cea16ead8134de8d9823bd/addons/test_website/__manifest__.py#L10-L14

But even `website` module in your case seems wrong?
It should be in `web_editor`, why in website? If you blame the commit which introduced the test already in this file, a very special reason is given: https://github.com/odoo/odoo/commit/d85ba585ec4642a507d24ab94606a65970c635f3
> As it is necessary to import it when the module is installed,
this test is moved to test_website as both modules are installed there.
(Also updated to ES6 import syntax + odoo-module style test)

Reply:
@rdeodoo Hello, yes sorry I started writing it in `web_editor`, then I moved it because the `MediaDialog` has a lot of service requirements (notably `web_unsplash`, which is only a dependency here), but then I figured I didn't need to setup all those services for this test because we only care about props, so I didn't use the `super` `setup`

I'll move it back to `web_editor` then